client/.env

```diff
@@ -0,0 +1 @@
+API_KEY = http://127.0.0.1:5000/
```

Comment:
should be `API_URL` as index.js has `baseURL: process.env.API_URL`

Reply:
Done